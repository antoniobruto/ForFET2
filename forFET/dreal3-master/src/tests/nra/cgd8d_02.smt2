;; https://github.com/dreal/dreal3/issues/337
(set-logic QF_NRA)
(set-info :precision 0.00001)

(declare-fun x () Real [-1, 2])
(declare-fun forall w () Real [-3.141593, 3.141593])
(assert (
        forall (
                (w Real)
        )
(or (< (cos w) 0.3) (<= 0.4 x))
))
(check-sat)
(exit)
