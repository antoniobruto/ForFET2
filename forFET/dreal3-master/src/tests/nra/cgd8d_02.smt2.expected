delta-sat with delta = 0.00001000000000000
